src/BarcodePushController.js

```diff
@@ -68,7 +68,7 @@ function (Okta, FactorUtil, FormController, FormType, RouterUtil, BarcodeView, F
 
     pollForEnrollment: function () {
       return this.model.doTransaction(function(transaction) {
-        return transaction.startEnrollFactorPoll(PUSH_INTERVAL);
+        return transaction.poll(PUSH_INTERVAL);
```

Comment:
I think we probably didn't clean this up the last time, but it looks like we can remove the "remove" function in this file (it's not doing anything anymore)